crowdfunding_compassion/migrations/12.0.1.0.1/pre-migration.py

```diff
@@ -0,0 +1,21 @@
+from openupgradelib import openupgrade
```

Comment:
This migration is for the module `sponsorship_switzerland`. I think your bug comes from that because sponsorship_switzerland will be updated before crowdfunding_compassion and therefore your migration is not working here. Move it to sponsorship_switzerland so that it executes before it reads the xml data files of the module sponsorship_Switzerland.

Reply:
I moved it to sponsorship_switzerland but I still get the same error. Maybe this is somehow related to the docker installation I have.